hwdb.d/60-keyboard.hwdb

```diff
@@ -283,11 +283,11 @@ evdev:name:Asus WMI hotkeys:dmi:bvn*:bvr*:bd*:svnASUS*:pn*:*
 evdev:name:Eee PC WMI hotkeys:dmi:bvn*:bvr*:bd*:svnASUS*:pn*:*
 evdev:name:Asus Laptop extra buttons:dmi:bvn*:bvr*:bd*:svnASUS*:pn*:*
  KEYBOARD_KEY_6b=f21                                    # Touchpad Toggle
- KEYBOARD_KEY_7c=f20                                    # Remap micmute to f20
+ KEYBOARD_KEY_7c=micmute
```

Comment:
The match lines impact 3 kernel drivers:

- drivers/platform/x86/asus-nb-wmi.c
- drivers/platform/x86/eeepc-wmi.c
- drivers/platform/x86/asus-laptop.c

All 3 use a sparse-keymap, only the asus-nb-wmi.c keymap contains a 0x7c scancode entry and the sparse-keymap code only allows applying mapping changes to scancodes which already have an entry.

So this change only impacts asus-nb-wmi.c which already maps 0x7c to micmute since 2019 and before that there was no mapping for this scancode (disallowing override from hwdb).

So I think this patch-series should drop this line. The dropping of now no longer mappings is probably best done in a follow-up patch rather then directly in this patch.